config/routes/find.rb

```diff
@@ -1,13 +1,15 @@
 constraints(FindConstraint.new) do
-  get "/", to: "find/application#index"
+  get "/", to: "find/search/locations#index", as: :find
 end
 
 namespace :find, path: "/find" do
-  get "/", to: "application#index"
+  scope module: :search do
+    resources :age_groups, path: "/age-groups"
```

Comment:
Do we need 
```suggestion
    resources :age_groups, path: "/age-groups", controller: "locations"
```
?

Reply:
You put the wrong controller :face_in_clouds:

Reply:
🫣